docs/samples/other-charts/stacked-bar-line.md

```diff
@@ -118,3 +118,13 @@ module.exports = {
   config: config,
 };
 ```
+
+## Docs
+* [Axes scales](../../axes/)
+  * [Common options to all axes (`stacked`)](../../axes/#common-options-to-all-axes)
```

Comment:
stacked part is unecesarry since they are common to all axes and now it looks like its only common to all stacked axes